src/components/DefaultEpic.tsx

```diff
@@ -56,51 +61,70 @@ const imageStyles = css`
-                on our lives. At a time when factual information is a necessity, we believe that
-                each of us, around the world, deserves access to accurate reporting with integrity
-                at its heart.
-            </p>
-            <p className={bodyStyles}>
-                Our editorial independence means we set our own agenda and voice our own opinions.
-                Guardian journalism is free from commercial and political bias and not influenced by
-                billionaire owners or shareholders. This means we can give a voice to those less
-                heard, explore where others turn away, and rigorously challenge those in power.
-            </p>
-            <p className={bodyStyles}>
-                We hope you will consider supporting us today. We need your support to keep
-                delivering quality journalism that’s open and independent. Every reader
-                contribution, however big or small, is so valuable.{' '}
-                <strong className={highlightWrapperStyles}>
-                    <span className={highlightStyles}>
-                        Support The Guardian from as little as £1 - and it only takes a minute.
-                        Thank you.
-                    </span>
-                </strong>
-            </p>
-            <div className={buttonWrapperStyles}>
-                <CallToAction
-                    url="https://support.theguardian.com/uk/contribute"
-                    linkText="Support The Guardian"
-                />
-                <img
-                    src="https://assets.guim.co.uk/images/acquisitions/2db3a266287f452355b68d4240df8087/payment-methods.png"
-                    alt="Accepted payment methods: Visa, Mastercard, American Express and PayPal"
-                    className={imageStyles}
-                />
-            </div>
-        </section>
-    );
+type Props = {
+    heading: string | undefined;
+    paragraphs: string[];
+    highlighted: string[];
+};
+
+type HighlightedProps = {
+    highlighted: string[];
+};
+
+type BodyProps = {
+    paragraphs: string[];
+    highlighted: string[];
 };
+
+const Highlighted: React.FC<HighlightedProps> = ({ highlighted }: HighlightedProps) => (
+    <strong className={highlightWrapperStyles}>
+        {' '}
+        {highlighted.map((highlightText, idx) => (
+            <span
+                key={idx}
+                className={highlightStyles}
+                dangerouslySetInnerHTML={{ __html: replacePlaceholders(highlightText) }}
+            />
+        ))}
+    </strong>
+);
+
+const Body: React.FC<BodyProps> = ({ highlighted, paragraphs }: BodyProps) => (
+    <>
+        {paragraphs.length > 0 &&
+            paragraphs.map((paragraph, idx) => (
+                <p key={idx} className={bodyStyles}>
+                    <span dangerouslySetInnerHTML={{ __html: replacePlaceholders(paragraph) }} />
```

Comment:
I've introduced an additional span here around the paragraph content to allow me to use `dangerouslySetInnerHtml`, since I also want to include children below (the `Highlighted` component appears within the same `<p>` tag). I'm interested to hear thoughts on alternative approaches here.

Reply:
Yeah this is annoying and I don't have any better solution I'm afraid :(.

Reply:
I don't have a better solution either. But for the sake of argument, if we absolutely had to stop that extra span from being added, I suppose we could move the `dangerouslySetInnerHTML` call back to the paragraph level and ensure the HTML we pass into it includes the paragraph text, as well as the result of passing the `Highlighted` component through ReactDOM's `renderToStaticMarkup` function, which we do use further upstream. But again, I personally wouldn't do that in this case as we can live with that span.